main.tf

```diff
@@ -49,9 +49,9 @@ locals {
   addon_values = yamlencode({
     admissionController = {
       rbac = {
+        create = local.addon_irsa["${local.addon.name}-admission-controller"].service_account_create
```

Comment:
Using `service_account_create` for `rbac.create` value might be misleading. I would change the variable name to align it with Helm chart values.